frontends/web/.eslintrc.json

```diff
@@ -21,7 +31,21 @@
     "arrow-spacing": "error",
     "space-infix-ops": "error",
     "react/no-unused-prop-types": "error",
-    "react/jsx-equals-spacing": ["error", "never"]
+    "react/jsx-equals-spacing": ["error", "never"],
+    "react/react-in-jsx-scope": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    "no-case-declarations": "off",
+    "react/no-children-prop": "off",
+    "prefer-const": "off",
```

Comment:
nice.
I randomly tested enabling this.. looks like we could enable some in the future once this is merged. (and maybe once we have less staging branches)

Reply:
Good find! This might be the reason why we saw the noise on.`make webtest` and needed this https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/2801#issuecomment-2217855467.